orttraining/orttraining/core/optimizer/bias_dropout_fusion.cc

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/optimizer/initializer.h"
+#include "orttraining/core/optimizer/bias_dropout_fusion.h"
+#include "core/graph/graph_utils.h"
+#include <deque>
+
+using namespace ONNX_NAMESPACE;
+using namespace ::onnxruntime::common;
+namespace onnxruntime {
+
+Status BiasDropoutFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level, const logging::Logger& logger) const {
```

Comment:
>ApplyImpl [](start = 26, length = 9)

this is a pretty big function - would it be worth splitting it into smaller ones for readability?